### DISCUSSION TITLE
does coc still work if you only have VSCode Insiders Edition installed?

### DISCUSSION THREAD

**Discussion by alok** [neoclide/coc.nvim#2709] [Q&A]
I'm considering switching to it but would like to know before installing a bunch of stuff (if possible).

**Comment by fannheyward** [comment]
coc doesn't need VSCode to work.

**Comment by chemzqm** [comment]
coc.nvim and it's extensions doesn't requires vscode to work, but you need to get familiar to vim before use this extension.
